src/infrastructure/transport/notes-api/index.ts

```diff
@@ -49,8 +50,9 @@ export default class NotesApiTransport extends AuthorizableTransport {
          */
         switch (status) {
           case 401:
-          case 403:
             return new UnauthorizedError(errorText);
+          case 403:
+            return new ForbiddenError(errorText);
```

Comment:
Why not `UnauthorizedError` ?

Reply:
because the user may be authorized but not have ehough permission

Reply:
if you don't have a permission, it means that you're not authorized to perform an action.

You're talking about authentication, not authorization.